build/mojblocks.min.js

```diff
@@ -2589,9 +2589,9 @@ if (mojblocksAccordion) {
 /*! no static exports found */
 /***/ (function(module, exports, __webpack_require__) {
 
-__webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/src/frontend.js */"./src/frontend.js");
-__webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/style.scss */"./style.scss");
-module.exports = __webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/style-gutenburg.scss */"./style-gutenburg.scss");
+__webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/src/frontend.js */"./src/frontend.js");
+__webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/style.scss */"./style.scss");
+module.exports = __webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/style-gutenburg.scss */"./style-gutenburg.scss");
 
```

Comment:
Not sure whats happening with this file. We should avoid having local stuff in it. Lets have a discussion.

Reply:
Also its not minfied

Reply:
Oh that's strange o.0 how come it's working like that?